<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClearSetDlg</class>
 <widget class="BaseDlg" name="ClearSetDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>690</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ClearSetDlg</string>
  </property>
  <widget class="QPushButton" name="quit_btn">
   <property name="geometry">
    <rect>
     <x>577</x>
     <y>593</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="buttons" stdset="0">
    <string>btn_dlg</string>
   </property>
  </widget>
  <widget class="QPushButton" name="save_btn">
   <property name="geometry">
    <rect>
     <x>447</x>
     <y>593</y>
     <width>116</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>680</width>
     <height>488</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <property name="tabs" stdset="0">
    <string>sub_border</string>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>数据</string>
    </attribute>
    <widget class="QGroupBox" name="clear_data_groupbox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>631</width>
       <height>418</height>
      </rect>
     </property>
     <property name="title">
      <string>清空数据</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="borders" stdset="0">
      <string notr="true">noborder_tit_check</string>
     </property>
     <widget class="QCheckBox" name="cali_ckbox">
      <property name="geometry">
       <rect>
        <x>455</x>
        <y>255</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>校准</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>213</y>
        <width>531</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>清空数据类型</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="startup_clear_rdbtn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>551</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>每日首次开机后清空</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>150</y>
        <width>240</width>
        <height>40</height>
       </rect>
      </property>
      <property name="wrapping">
       <bool>false</bool>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="sample_ckbox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>255</y>
        <width>201</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>样本</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="qc_ckbox">
      <property name="geometry">
       <rect>
        <x>264</x>
        <y>255</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>质控</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="timing_clear_rdbtn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>105</y>
        <width>571</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>定时清空</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="complete_ckbox">
      <property name="geometry">
       <rect>
        <x>264</x>
        <y>350</y>
        <width>181</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>已完成</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>62</x>
        <y>308</y>
        <width>531</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>清空状态</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pending_ckbox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>350</y>
        <width>201</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>待测</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>位置设置</string>
    </attribute>
    <widget class="QGroupBox" name="clear_seat_groupbox">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>651</width>
       <height>311</height>
      </rect>
     </property>
     <property name="title">
      <string>清空位置设置</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <property name="borders" stdset="0">
      <string notr="true">noborder_tit_check</string>
     </property>
     <widget class="QCheckBox" name="qc_complete_ckbox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>105</y>
        <width>581</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>质控完成清空质控品位置</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cali_complete_ckbox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>581</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>校准完成清空校准品位置</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseDlg</class>
   <extends>QWidget</extends>
   <header>shared/basedlg.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>quit_btn</sender>
   <signal>clicked()</signal>
   <receiver>ClearSetDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>724</x>
     <y>487</y>
    </hint>
    <hint type="destinationlabel">
     <x>697</x>
     <y>397</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
